src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,18 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => {
+  if (!comments) {
```

Comment:
You don't need to render the `CommentList` component at all if you don't have any comments